pkg/kubelet/metrics/metrics.go

```diff
@@ -917,6 +928,16 @@ var (
 			StabilityLevel: metrics.ALPHA,
 		},
 	)
+
+	ContainerAlignedComputeResources = metrics.NewCounterVec(
```

Comment:
Can you put it next to `CPUManagerPinningRequestsTotal` metric? They seems to be related. Also what is the difference between `NewCounterVec` and `NewCounter`?

Reply:
sure, moving.
Regarding Counter vs CounterVec:
CounterVec bundles a set of Counters that all share the same Description. We use it because we want to count the same value partitioned by various dimensions: scope and boundary.